src/qibo/abstractions/circuit.py

```diff
@@ -553,6 +545,8 @@ def set_parameters(self, parameters: Union[Dict, List]):
         """
         if isinstance(parameters, (list, tuple)):
             self._set_parameters_list(parameters, len(parameters))
+        elif isinstance(parameters, self.param_tensor_types):
```

Comment:
So this opens the possibility to use tensors as parameters, shall we update the docs?

Reply:
This is not a new feature, I think it exists in our current master, here I just changed slightly how the type of `parameters` is checked. Currently in the [How to use parametrized gates?](https://qibo.readthedocs.io/en/stable/advancedexamples.html#how-to-use-parametrized-gates) example in the docs we mention that a `np.array` or a `tf.Tensor` can be used in place of the flat list in `circuit.set_parameters` and we have an example with the variational layer. Do you think we should update something else regarding this?

Reply:
Right. I think we can update the info for `parameters` in the docstring of this function.